apps/frontend/src/libs/components/app/app.tsx

```diff
@@ -71,6 +90,13 @@ const App = (): JSX.Element => {
 					<Table<Person> columns={mockTableColumns} data={mockTableData} />
 				</>
 			)}
+			<Select<SelectData, number>
+				control={control}
+				isMulti
+				name="options"
+				options={options}
```

Comment:
```suggestion
				options={[]}
```

Reply:
Liza asked me to leave some data to test, but okay.